crates/linker/src/pe.rs

```diff
@@ -408,12 +408,12 @@ fn mmap_mut(path: &Path, length: usize) -> MmapMut {
     unsafe { MmapMut::map_mut(&out_file).unwrap_or_else(|e| internal_error!("{e}")) }
 }
 
+/// Find the place in the executable where the thunks for our dummy .dll are stored
 #[allow(dead_code)]
-fn redirect_dummy_dll_functions(
-    app: &mut [u8],
+fn find_thunks_start_offset(
+    executable: &[u8],
     dynamic_relocations: &DynamicRelocationsPe,
-    function_definition_vas: &[u64],
-) {
+) -> usize {
```

Comment:
I split this into two functions, plus some glue for the tests. This turns out to be useful later when we want to gather data from the `dynhost.exe`, write that to disk, and then when linking we only write to the output, we don't read from the `.exe` we're building.